examples/e-commerce/src/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "instantsearch-ecommerce",
```

Comment:
The sort name must be at most 12 characters. For the other e-commerce demos, we went for:

```suggestion
  "short_name": "E-commerce",
```